go/tasks/pluginmachinery/utils/error_collection.go

```diff
@@ -0,0 +1,19 @@
+package utils
```

Comment:
i think maybe plugins doesn't use this but can we move this to https://github.com/lyft/flytestdlib/blob/master/errors/error.go eventually?

Reply:
yes i agree we should move, its replicated in a few places. I will create an issue